sdk/monitor/azquery/autorest.md

```diff
@@ -1,25 +1,19 @@
 ## Go
 
 ``` yaml
-title: MonitorQueryClient
-description: Azure Monitor Query Go Client
-generated-metadata: false
-
+title: Logs Query Client
```

Comment:
autorest help claims `title` overrides the client name but actually it doesn't appear to have any effect and we have `override-client-name` as well, which also appears to do nothing. 🤔 should we just omit this?

Reply:
`override-client-name` doesn't do anything, so removed in [d0c65c0](https://github.com/Azure/azure-sdk-for-go/pull/21965/commits/d0c65c005478be252e03a390bdc8d4ded5ffe04a)